### ISSUE TITLE
klaro methods documentation?

### ISSUE THREAD

**Issue by Genenenenaam** [KIProtect/klaro#67]
**Would be nice to have some documentation about the methods that klaro offers.**

Looking around in the code is see methods that seem useful, but without documentation it's hard to see their use.

`klaro.getManager().resetConsent();`
`klaro.show();`
`klaro.deleteCookie();`

**Comment by enterframe** [comment]
+1

**Comment by adewes** [comment]
Yes we're working on a new and greatly extended documentation site, I'll post it here as soon as it's online!
